### DISCUSSION TITLE
已老实，再也不丢了

### DISCUSSION THREAD

**Discussion by kidmil1** [BurnHair/docs#109] [分享讨论]
<img width="226" alt="PixPin_2024-06-20_00-28-26" src="https://github.com/BurnHair/docs/assets/82021272/c9c005d8-0b02-483a-9888-21819a732071">

**Comment by dao-zero** [comment]
我从开始到现在全是亏，最好的就是不赚不亏，真的想死

**Comment by ENIAC1729** [comment]
为什么我这计算的是5但是突然卡了下就变成1了
![图片](https://github.com/BurnHair/docs/assets/108004193/49d3cee6-9ec8-4629-b1c1-eda9c1b235a3)

**Reply by BennyThink** [reply]
你忘记改 `userId` 了，改成 `5677` 再试试
